data/remotes.d/fwupd.conf

```diff
@@ -0,0 +1,6 @@
+[fwupd Remote]
+
+# this remote provides metadata shipped with the fwupd package
+Enabled=true
+Type=local
+File=@datadir@/fwupd/remotes.d/fwupd/metadata.xml
```

Comment:
I thought about this; I don't know how to specify the output file name when calling gzip using meson. Ideas welcome.